gitbook-docs/market/cmd-reference/statement.md

```diff
@@ -1,75 +1,104 @@
+`prepare_statement.py` and `statement_tools.py`.
 
+Make sure you've [signed up](user.md) before continuing.
 
-### Prepare Statement
+## Preparing circuit as a statement
 
-Statements/circuits generated by the zkLLVM compiler need to have additional metadata attached to them before they can be published on the proof market.
+Circuits are stored as a *statement* structure on the Proof Market.
+Statement examples can be found in `./example/statements/` directory.
+Attach all additional metadata to circuits generated by the zkLLVM compiler
+before publishing them on the Proof Market.
 
-**Usage**
+### Usage
 
-```
-python3 prepare_statement.py -h 
-```
+To see all parameters that can be passed to the script, run it with the `-h` option:
 
-Output
+```console
+$ python3 prepare_statement.py -h 
 
-```
-usage: prepare_statement.py [-h] -c CIRCUIT -o output file -n NAME -t TYPE
+usage: prepare_statement.py [-h] -c CIRCUIT -o output_file_path -n NAME -t \
+       statement_type (--private | --public)
 
 options:
   -h, --help            show this help message and exit
   -c CIRCUIT, --circuit CIRCUIT
-                        zkllvm compiler output
-  -o output file, --output output file
-                        output file
-  -n NAME, --name NAME  name
-  -t TYPE, --type TYPE  statement type: placeholder-zkllvm or placeholder-vanilla
-
+                        Path to a zkLLVM circuit (*.ll)
+  -o output_file_path, --output output_file_path
+                        Output file
+  -n NAME, --name NAME  Statement name that will be shown on the Proof Market
+  -t statement_type, --type statement_type
+                        Statement type: placeholder-zkllvm or placeholder-vanilla
+  --private             Make a statement private: only accessible by ID
+  --public              Make a statement public: visible on the Proof Market
 ```
 
-Ex:
+Please note that `--private` and `--public` parameters are mutually exclusive:
+* With `--private`, the statement will be accessible only by its ID.
+  For experiments and development purposes, make your statements private.
+* With `--public`, the statement will be openly listed on the Proof Market.
 
-```
-python3 prepare_statement.py -c arithmetics_example.ll -n arithmetic -t placeholder-zkllvm -o arithmetic.json
+### Example
+
+```bash
+python3 prepare_statement.py -c arithmetics_example.ll -n arithmetic -t placeholder-zkllvm -o \
+arithmetic.json --private
 ```
 
-### Publish Statement
+## Publishing statement
 
-Statements/Circuits can be published to the proof market once modified into the required format. See [prepare statement above](statement.md#prepare-statement).
+Statements/circuits can be published to the Proof Market once modified into the required format.
+See the [preparing statement](statement.md#preparing-circuit-as-a-statement) section above.
 
-#### Usage
+### Usage
+
+To publish a statement, you'll need to pass its JSON statement description in a file:
```

Comment:
no-no, there's no JSON description of a statement. A JSON is the statement itself. The file. But it doesn't matter that it's JSON.

Reply:
Removed this for now